src/script/E2EIdentity/E2EIdentityVerification.ts

```diff
@@ -62,7 +62,20 @@ export async function getDeviceIdentity(
   return undefined;
 }
 
-export async function getConversationState(groupId: string) {
+// TODO: replace implementation with CoreCrypto once it has user verification method
+export async function getUsersVerificationState(groupId: string, userIds: QualifiedId[]) {
+  return userIds.map(userId => ({
+    userId,
+    state: Math.random() > 0.5 ? MLSStatuses.VALID : MLSStatuses.EXPIRED,
+  }));
+}
```

Comment:
For now, CoreCrypto, doesn't return the verification state of users. This is going to be implemented in the next version of it. 

For now, we just randomize the value in order to see the different states in the UI